acme/acme/errors.py

```diff
@@ -49,7 +49,7 @@ def __init__(self, response, *args, **kwargs):
 
     def __str__(self):
         return ('Server {0} response did not include a replay '
-                'nonce, headers: {1}'.format(
+                'nonce, headers: {1}\n(This may be a service outage)'.format(
```

Comment:
nit: I think we should remove the newline. None of the other exceptions in the `acme` library contain a newline and I think it is a bit odd to have it there. We don't really want the newline in log files for example.

Also, we could potentially catch `MissingNonce` errors at a higher level in the client and provide a better error message. Ideally, we'd do this at the lowest level in the client where we can catch the exceptions, however, its thrown at multiple places and we're worried about all of them (maybe we only really care about the first time we expect a nonce from the server?) then we could handle this in the `_handle_exception` function in `main.py`.